src/app.jsx

```diff
@@ -134,8 +134,27 @@ class Application extends React.Component {
                 });
     }
 
+    isContainerCheckpointPresent(id, system) {
```

Comment:
Oh I see, so you actually verify that it is present for each and every container. So no need to bother with checking of cgroupsV2. (But still there should be test that the button is not there, when on cgroupsV2 and once it will be available, we will notice in tests :) )

Reply:
This checks whether a checkpoint is present for the container (i.e. the container can be restored), not whether the container can be checkpointed.